### PR TITLE
[W5.11][T12-1] Chuang Qin Kai

### PR DESCRIPTION
**Pull request by cqinkai**

### CODE REVIEW

**Comment by Geraldcdx** [comment]
Easy to read code that you added. But best if you could commit incrementally so we can track the changes better! Good job

File: src/seedu/addressbook/data/person/DateOfBirth.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+/**
+ * Represents a Person's date of birth in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidDob(String)}
+ */
+public class DateOfBirth {
+
+    public static final String EXAMPLE = "12-09-1998";
+    public static final String MESSAGE_DOB_CONSTRAINTS = "Person dates of birth should be 3 numbers separated by '-'";
+    public static final String DOB_VALIDATION_REGEX = "\\d+-\\d+-\\d+";
```
Comment (devamanyu) [inline src/seedu/addressbook/data/person/DateOfBirth.java:13]: Try to see how you can improve this regex to handle months and day constraints.

File: src/seedu/addressbook/storage/AddressBookDecoder.java [hunk 2]
```diff
@@ -11,12 +11,7 @@
 
 import seedu.addressbook.data.AddressBook;
 import seedu.addressbook.data.exception.IllegalValueException;
-import seedu.addressbook.data.person.Address;
-import seedu.addressbook.data.person.Email;
-import seedu.addressbook.data.person.Name;
-import seedu.addressbook.data.person.Person;
-import seedu.addressbook.data.person.Phone;
-import seedu.addressbook.data.person.UniquePersonList;
+import seedu.addressbook.data.person.*;
```
Comment (devamanyu) [inline src/seedu/addressbook/storage/AddressBookDecoder.java:14]: Avoid doing batch import as it violates the coding standard.

**Review by devamanyu — APPROVED** [review]
Good implementation of adding a DOB field. Consider updating the user guides with clear instructions to facilitate usage. 

@Geraldcdx correctly points out that commits can be made more incrementally to ensure better tracking.

Some comments added on coding styles. Please close this PR after reading them.

File: src/seedu/addressbook/data/person/DateOfBirth.java (hunk 1 above)
Comment (cqinkai) [reply src/seedu/addressbook/data/person/DateOfBirth.java:13]: Is this a possible way? 
public static final String DOB_VALIDATION_REGEX = "\\d{2}-\\d{2}-\\d{4}";

**Pull request closed without merging**